### PR TITLE
Fe create sign in component

### PR DESCRIPTION
**Pull request by edyhtan**
- Added Login Page
- Added Sign in Page

Sign-in page screenshot attached. The login page looks very identical to this. 

The style is preliminary. 

[
SignIn
](url)

### CODE REVIEW

File: client/src/pages/Signup.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        const empty = event.target.value.length == 0 
+        this.setState({
+            email: event.target.value,
+            emailError: empty || properEmail,
+            emailWarning: empty ? warningMsg.emptyFieldError : (properEmail ? warningMsg.emailFormatError : "")
+        })
+    }
+
+    onConfirmPWChange = (event) => {
+        const lengthCheck = event.target.value.length < 6
```
Comment (bonnieli) [inline client/src/pages/Signup.js:78]: you can save `event.target.value` in a variable so you don't need to keep repeating `event.target.value`

**Review by bonnieli — COMMENTED** [review]
@banh0006 please translate these components into functional components, and utilizing React hooks instead so that we can practice using the most up to date features of React

**Comment by banh0006** [comment]
> @banh0006 please translate these components into functional components, and utilizing React hooks instead so that we can practice using the most up to date features of React

Yes, for sure. I prefer using React Hooks It's better than the class component. So I will translate all components into functional components in a new branch after we merge this pull request.

**Review by banh0006 — APPROVED** [review]

**Comment by bonnieli** [comment]
@banh0006 sounds good that's a good idea. you can feel free to merge this in and refactor it in a separate PR :)

**Review by bonnieli — APPROVED** [review]

**Pull request merged**